platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -206,6 +210,25 @@ public Path getExportFile() {
     return contexts.values();
   }
 
+  private Properties getImportedCustomSystemProperties(Map<String, Object> metadata) {
+    if (version.equals(MigrationContextImpl.SYSTEM_PROPERTIES_MISSING_VERSION)) {
+      Properties importedProps = new Properties();
+      try {
+        importedProps.load(
+            contexts
+                .get(MigrationContextImpl.PLATFORM_MIGRATABLE_ID)
+                .getEntry(MigrationContextImpl.METADATA_CUSTOM_SYSTEM_PROPERTIES_PATH)
+                .getInputStream()
+                .orElseThrow(IOException::new));
+      } catch (IOException e) {
+        LOGGER.warn("Could not import custom system properties.");
```

Comment:
❗️ This should fail the import and the exception should be logged to help debug.